### DISCUSSION TITLE
Missing complex conjugation in Quantum Circuits / 2.1 Inner products

### DISCUSSION THREAD

**Discussion by gavinking** [Qiskit/.github#30] [Documentation]
Folks, [section 2.1](https://learn.qiskit.org/course/basics/quantum-circuits#quantum-16-0) of Basics of Quantum Information is missing *lots* of complex conjugation.

For example, equations (1) and (2) are incorrect, as are equations defining conjugate symmetry and conjugate linearity of the inner product.

If someone could do me the favor or pointing me to the source code for this page in the correct git repo, I can fix it and send a pull request.

Thanks

**Comment by kdk** [comment]
Hi @gavinking , thanks for pointing this out. The new textbook is maintained at https://github.com/Qiskit/platypus/ so please open an issue there. 

(As an aside, these equations look okay to me, but it's entirely possible I'm missing something. $\overline{\alpha_n}$ can be used (along with $\alpha_n^*$) to represent complex conjugation, but I also don't see that called out in the text, which may be a bug in and of itself.)

**Reply by gavinking** [reply]
OK, so I checked the source code here:

<https://github.com/Qiskit/platypus/blob/main/notebooks/basics/quantum-circuits.ipynb>

and the equations are indeed correct.

But if you compare those correct equations to how they're rendered here:

<https://learn.qiskit.org/course/basics/quantum-circuits#quantum-16-0>

you'll see the difference.

So perhaps it's something to do with the code which transforms the Python notebooks to static html?

**Comment by gavinking** [comment]
I opened an issue: https://github.com/Qiskit/platypus/issues/2041
